addons/etcd/scripts/switchover-without-candidate.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -exo pipefail
+
+leader_endpoints=${KB_CONSENSUS_LEADER_POD_FQDN}:2379
```

Comment:
KBSwitchoverConsensusLeaderPodName is deprecated, use KBSwitchoverLeaderPodFqdn instead

Reply:
solved